exercises/kindergarten-garden/uKindergartenGardenTest.pas

```diff
@@ -0,0 +1,145 @@
+unit uKindergartenGardenTest;
+
+interface
+uses
+  DUnitX.TestFramework, uKindergartenGarden;
+
+type
+  [TestFixture]
+  TPartialGardenTest = class(TObject)
+  private
+    procedure CompareStringArrays(ExpectedArray, ActualArray : TArray<string>);
+  public
+    [Test]
+//  [Ignore('Comment the "[Ignore]" statement to run the test')]
```

Comment:
add 2 spaces after `//` so that the rest of the line after `//` is indented 2 spaces.